mbuild/formats/lammpsdata.py

```diff
@@ -41,6 +41,9 @@ def write_lammpsdata(
     use_urey_bradleys=False,
     use_rb_torsions=True,
     use_dihedrals=False,
+    sigma_conversion_factor=None,
+    epsilon_conversion_factor=None,
+    mass_conversion_factor=None,
```

Comment:
I agree that more info is required. There are two main issues: 1) Since Foyer XML files follow the OpenMM format, they assume a specific set of units (KJ, nm, etc). For users who wish to specify their XML files **in reduced units**, there currently isn't an option for using a different unit set. 2) In addition to the unit set assumed by OpenMM, Parmed does additional conversions in the background (e.g., KJ to KCAL and NM to Ang). If a user specifies a Foyer XML file in a set of units, there should **not** be background conversions. Hence why this PR reverses the background conversion.

For the ethane test case, if you take a look at foyer/foyer/forcefields/xml/oplsaa.xml, all units are specified according to OpenMM (e.g. KJ) but when the LAMMPS data file is outputted using the old writer, the units are converted to KCAL, ANG, etc. So, the background conversions are necessary for this test case because **the XML file follows the assumed set of units used by OpenMM**. 

Moving forward, I think it might be best **to allow the user to specify any unit set as an input to the Foyer XML**. Therefore, if the background conversions are reversed (with this PR), then the same unit set will be outputted in the LAMMPS data file. This also relates back to the issue when the conversion factors are not specified that it will take the largest values of sigma, epsilon, and mass and **will assume they are in the same unit set as the unit set for the XML file**. So, everything is consistent, even though the new lammps writer will give different reduced values (e.g., k will be different as you pointed out)

Thoughts?